code/game/objects/structures/burrows.dm

```diff
@@ -554,6 +554,17 @@ percentage is a value in the range 0..1 that determines what portion of this mob
 //This proc handles creation of a plant on this burrow
 //It relies on the plant seed already being set
 /obj/structure/burrow/proc/spread_plants()
+	if(istype(plant, /datum/seed/wires))		//hivemind wireweeds handling
+		if(locate(/obj/effect/plant) in loc)
+			return
+
+		var/obj/structure/hivemind_machine/node/hivemind_node = locate() in world
```

Comment:
is there any way to have this stored somewhere that isn't world? locate() in world isn't nearly as performant as a var access, AFAIK (not entirely sure though)

Reply:
you are completely correct